inspirehep/modules/multieditor/exceptions.py

```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of INSPIRE.
+# Copyright (C) 2017 CERN.
+#
+# INSPIRE is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# INSPIRE is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with INSPIRE. If not, see <http://www.gnu.org/licenses/>.
+#
+# In applying this license, CERN does not waive the privileges and immunities
+# granted to it by virtue of its status as an Intergovernmental Organization
+# or submit itself to any jurisdiction.
+
+from __future__ import absolute_import, division, print_function
+
+
+class InvalidActions(Exception):
```

Comment:
```python
class InvalidActions(Exception):
    """Invalid action."""

    def __init__(self):
        """Init with message."""
        super(InvalidActions, self).__init__("The actions that were provided are invalid")
```

Reply:
Don't you think that this strips out usability from the exception? For example, you would not be able to do something like:
```python
if action not in available_actions:
    raise InvalidActions('Action "%s" is not a valid action, valid actions are %s' % (action, available_actions))
```

That would help a lot in debugging (as you can see the actual action value passed and the allowed ones).

Reply:
I agree though that if you directly use a superclass method, you must use super (it never hurts, but it solves possible issues).